Assets/Scripts/HexVertices.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// A list of Hex types supported by this package.
+/// </summary>
+public enum HexTypes
+{
+    FlatTop,
+    PointyTop
+}
+
+/// <summary>
+/// HexVertices defines arrays of Vector3s representing the vertices of Hexes.

```

Comment:
Nit: You used `Vector2[]` in a previous doc comment. Be consistent and use `Vector3[]`.

Reply:
Changed